pysat/tests/test_orbits.py

```diff
@@ -1,11 +1,26 @@
 from dateutil.relativedelta import relativedelta as relativedelta
+from nose.tools import raises
 import numpy as np
-
 import pandas as pds
 
 import pysat
 
 
+class TestOrbitsUserInterface():
+
+    @raises(NameError)
+    def test_orbit_wo_info(self):
+        self.testInst = pysat.Instrument('pysat', 'testing',
+                                         clean_level='clean',
+                                         orbit_info=info, update_files=True)
```

Comment:
Looking over coveralls, this doesn't add anything to the coverage.  I think the name error is generated deeper in one of the packages, not in our code.  Should we keep it?

Reply:
Went ahead and pulled this extra test out.

Reply:
Went ahead and pulled this extra test out.